tests/cases/fourslash/importNameCodeFixNewImportNodeModules5.ts

```diff
@@ -1,25 +0,0 @@
-/// <reference path="fourslash.ts" />
-
-//// [|f1/*0*/('');|]
-
-// @Filename: package.json
-//// { "dependencies": { "package-name": "latest" } }
-
-// @Filename: node_modules/package-name/node_modules/package-name2/bin/lib/libfile.d.ts

```

Comment:
Don't think we should have been getting imports from here -- it's fishy to directly import a dependency of a dependency.

Reply:
Agreed it's fishy, but this tests seems it's a regression from something that can be done.

I recommend leaving the test in place to check for any regressions.

Reply:
I deleted it file *because* the test started failing, so it's an intentional regression -- I really think we shouldn't have been doing this.
This test was added in #17302 -- @minestarks is this OK to delete?

Reply:
yep seems fine to quit doing this. You could just change the expectation of the test case though...